tools/msgdis.py

```diff
@@ -404,3 +404,23 @@ def extract_all_text(text_out, staff_text_out):
 
         with open(staff_text_out, "w", encoding="utf8") as outfile:
             outfile.write(out.strip() + "\n")
+
+
+def main():
+    from argparse import ArgumentParser
```

Comment:
why here and not with the other imports?

Reply:
why not, it's not needed elsewhere

Reply:
It doesn't save any runtime, especially now that msgdis.py is only run by itself and not as a library (although even then argparse was already imported). And generally Python programs import everything globally, not per-function, unless there's a strong reason to break from that